src/main/java/oshi/jna/platform/linux/Udev.java

```diff
@@ -0,0 +1,168 @@
+/**
+ * Oshi (https://github.com/dblock/oshi)
+ *
+ * Copyright (c) 2010 - 2016 The Oshi Project Team
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * enrico[dot]bianchi[at]gmail[dot]com
+ *    com.github.dblock - initial API and implementation and/or initial documentation
+ */
+package oshi.jna.platform.linux;
+
+import com.sun.jna.Library;
+import com.sun.jna.Native;
+import com.sun.jna.NativeLibrary;
+import com.sun.jna.Pointer;
+import com.sun.jna.PointerType;
+import com.sun.jna.Structure;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Interface for talking with Udev.
+ *
+ * @author enrico[dot]bianchi[at]gmail[dot]com
+ */
+public interface Udev extends Library {
+
+    public static final NativeLibrary JNA_NATIVE_LIB = NativeLibrary.getInstance("udev");
+    public static final Udev INSTANCE = (Udev) Native.loadLibrary("udev", Udev.class);
+
+    public static final class BlockDevStats extends Structure {
+
+        public BlockDevStats(long read_ops,
```

Comment:
It's a mistake, the code that populate this structure isn't the JNA library, because the `udev_device_get_sysattr_value();` syscall returns always a string. So, I converted in a POJO the structure but I don't moved in a correct place